src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,29 @@
+.catalog {
+  --column-count: 1;
+
+  display: grid;
+  box-sizing: border-box;
+  padding: 50px 40px;
+  grid-template-columns: repeat(auto-fill, 200px);
```

Comment:
The `grid-template-columns` property is set to `repeat(auto-fill, 200px)`. This means that as many columns of 200px will be created as can fit in the container. However, the `--column-count` variable is not used here. If you intended to use this variable to control the number of columns, you should use it in this property.

Reply:
<img width="637" alt="image" src="https://github.com/user-attachments/assets/7208ed62-f6da-40f4-b946-e5443e8889ed">